src/Code.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withTheme } from 'styled-components'
+import SyntaxHighlighter, { registerLanguage } from "react-syntax-highlighter/prism-light"
+import defaultTheme from 'react-syntax-highlighter/styles/prism/atom-dark'
+import javascript from 'react-syntax-highlighter/languages/prism/javascript'
+import elixir from 'react-syntax-highlighter/languages/prism/elixir'
```

Comment:
JS I think makes sense as a default (since this is all built with React/JS), but elixir seems a little opinionated. I'd like the languages to be configurable somehow, but not sure the best way to approach that with the way `react-syntax-highlighter` works...

Reply:
I just randomly added a second language, just wanted to choose a second language.